com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 - Fixed NetworkLists not populating on client. NetworkList now uses the most recent list as opposed to the list at the end of previous frame, when sending full updates to dynamically spawned NetworkObject. The difference in behaviour is required as scene management spawns those objects at a different time in the frame, relative to updates. (#2062)
 - Fixed NetworkList Value event on the server. PreviousValue is now set correctly when a new value is set through property setter. (#2067)
 - Fixed NetworkList issue that showed when inserting at the very end of a NetworkList (#2099)
+- Fixed Owner-written NetworkVariable infinitely write themselves (#2109)
```

Comment:
Nit-pick: Could you migrate this up towards the top of the fixed list?  It makes it easier to look over if the order is descending (I know... I see a few in there that are ascending which we should fix too).

Reply:
updated